R/load.R

```diff
@@ -146,22 +146,22 @@ renv_load_action <- function(project) {
   if (!interactive())
     return("load")
 
-  # if this project already contains an 'renv' folder, assume it's
-  # already been initialized and we can directly load it
-  renv <- renv_paths_renv(project = project, profile = FALSE)
-  if (dir.exists(renv))
-    return("load")
-
```

Comment:
I think this check should ?probably? come after the one that checks for "are we autoloading"?

I also wonder if the condition should gain another check: Is there a project-specific library? Because if you `git clone` someone else's renv project, the `renv/` directory will be there but you can't really say that the project has been initialized. It's possible I'm wrong about this, because I'm still a bit foggy on the different states and verbs.

I can also imagine that these two matters interact with each other. I.e. if the condition gets modified, does that change the need to reorder these?

Reply:
Relocated.